tests/sqlparser_clickhouse.rs

```diff
@@ -336,6 +336,13 @@ fn parse_create_table() {
     );
 }
 
+#[test]
+fn parse_limit_by() {
```

Comment:
Can you also please add a test for the multi-column form (e.g. `LIMIT BY asset, class` or similar`)?

Reply:
Added